### ISSUE TITLE
Implement KYC

### ISSUE THREAD

**Issue by ken-de-moor** [threefoldtech/threefold_connect#299]
- Use Shufti
- Implement inside TF Connect

**OPEN QUESTIONS TO BE DISCUSSED**

1) Will it be implemented in the same way as phone verification under settings? Where it is not mandatory, but can be asked by sites to be accessed?
**=> Yes**
2) Will the info only be available after the user shares it, like phone number, or will we save the info or be able to access it ourselves?
**=> Yes**
3) Will we separate all user info in separate data ? ( Will require new tab/layout ) or will it just be one shared data object "Personal Identifying Data" or something, and sharing it will immediately share all the identifying data with that site?
**=> Yes**
4) What is the actual goal of the identification? Are we to lock all wallet access behind having to verify? Is it for all new accounts? Is it only for sites that request the identifier ( like mail/phone now )?
**=> Just save it, for now no wallet or registration access tied to it.**

Basically we need more specs than just "implement KYC"

What data should be there, what should require which data, and how do other apps or the TF team access this data.

**Comment by ken-de-moor** [comment]
Currently issue with redirect after verify, screen remains frozen. Will be fixed

**Comment by ken-de-moor** [comment]
BLocking issue was resolved, another blocking issue found.

You get multiple callbacks from shufti, and you lose context.
